scripts/test.canisters.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
This is the only script with multiple dots in the name. What about calling it `test-canisters.sh`?

Reply:
Yes, good point.